models/login_source.go

```diff
@@ -413,7 +413,7 @@ func LoginViaLDAP(user *User, login, password string, source *LoginSource, autoR
 	sr := source.Cfg.(*LDAPConfig).SearchEntry(login, password, source.Type == LoginDLDAP)
 	if sr == nil {
 		// User not in LDAP, do nothing
-		return nil, ErrUserNotExist{0, login, 0}
+		return nil, ErrUserNotExist{0, login, 0, -1}
 	}
```

Comment:
Why -1? I think it is better user type individual as you can't login with organization. Same in most other places.